src/Sentry/Platforms/Native/Sentry.Native.targets

```diff
@@ -85,11 +85,25 @@
   <!-- Build the Sentry Native SDK (this only runs on local machines because in CI we expect the SDK to be
        built already on each native platform and fetched for the final .net build. -->
   <Target Name="_BuildSentryNativeSDK"
-    BeforeTargets="DispatchToInnerBuilds;BeforeBuild"
-    Condition="'$(_SentryIsNet8OrGreater)' == 'true' and '$(CI)' != 'true'"
```

Comment:
The condition needs to be added to the new one's as `'$(_SentryIsNet8OrGreater)' == 'true'` does not resolve due to TF being removed.